src/build_info.cpp

```diff
@@ -558,11 +567,25 @@ list_formatter video_settings_report_internal(const std::string& heading = "")
 	const auto& current_driver = video::current_driver();
 	auto drivers = video::enumerate_drivers();
 
+	const auto& dpi = video::get_dpi();
+	std::string dpi_report;
+
+	dpi_report = dpi.first == 0.0f || dpi.second == 0.0f ?
+				 "<unknown>" :
```

Comment:
Is this specifically because the string `0x0` looks more like hexadecimal than like a pair of numbers?

Reply:
No, it's simply a display convention I already used in previous versions of this code (cf 1.16) for nonsense values like this.